days/day-09/solutions/day09.zig

```diff
@@ -0,0 +1,115 @@
+const std = @import("std");
+const mem = std.mem;
+const io = std.io;
+const d_print = std.debug.print;
+
+const MAX_FILE_SIZE = 20_000;
+const MAX_VALUES = 1500;
+
+const CrackError = error {
+    outOfMemory,
+    didNotFind,
+};
+
+const XMASCracker = struct {
```

Comment:
Wow, looks like suspicious software

Reply:
All your bases belongs to us. Merry christmas! :santa: